src/docs/meta/relay/best-practices.mdx

```diff
@@ -46,21 +108,15 @@ limits:
   max_thread_count: 8
   # Number of concurrent upstream requests for event submission
   max_concurrent_requests: 500
-cache:
-  # How many events relay buffers before it starts dropping new events
-  event_buffer_size: 10000
-  # How long a project configuration can still be used after having expired (in seconds).
-  # This might help when the upstream is unreachable (e.g., in case of network issues).
-  project_grace_period: 3600
-  # The cache timeout for buffered events before dropping them (in seconds)
-  event_expiry: 1800
```

Comment:
I'm on board with removing these. However, I think we should call out these two:

1. `event_buffer_size`, since this depends on the event volume. Of course, until we have better buffering in place, which can still take some time.
2. `event_expiry`. Even if we have a good default, it can be a conscious business decision to change the value.

Reply:
So does "call out" mean that you want to keep them?
Or move them to a separate section?

Reply:
What if we mention them in the table above under something like "settings"? And then you can keep them in the example here, for copy paste, but with the default values.

Reply:
The table above is for our "recommendations", but what I think you want is a description of some common settings that might be relevant for scaling (and again, default values should cover most of the normal cases). I don't think that the table is a right place for them, then.